scripts/ci/test_reborn_pr_test_plan.py

```
@@ -513,6 +873,70 @@ def test_agent_guidance_does_not_mask_a_real_lane_in_the_same_pr(self) -> None:
+        """Negative probe: widening IGNORED_PREFIXES must not open the gate.
+
+        The build-input probe was `Dockerfile` until #7084 gave that path a
+        decision (see `test_container_and_hook_inputs_are_owned_by_static_gates`);
+        `Makefile` is the genuinely undecided repo-root build input that keeps
+        this arm exercised, matching `test_unclassified_build_input_fails_fast`.
+        """
+        for path in ("Makefile", "claude/rules/x.md", ".claudeignore"):
+            with self.subTest(path=path):
+                with self.assertRaisesRegex(
+                    ValueError, "unclassified pull-request path"
+                ):
```

Ruff (0.16.0)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)